packages/core/src/kbjwt.ts

```diff
@@ -9,23 +15,26 @@ export class KBJwt<
   // Checking the validity of the key binding jwt
   // the type unknown is not good, but we don't know at this point how to get the public key of the signer, this is defined in the kbVerifier
   public async verifyKB(values: { verifier: KbVerifier; payload: JwtPayload }) {
+    if (!this.header || !this.payload || !this.signature) {
+      throw new SDJWTException('Verify Error: Invalid JWT');
+    }
+
     if (
-      !this.header?.alg ||
+      !this.header.alg ||
```

Comment:
Fine for now, but I think having an `assertHeaderClaim` method will help a lot here to make the code more readable and we can also return messages that tell the user on which property it failed.

like this: https://github.com/berendsliedrecht/sd-jwt-ts/blob/a1fb5fad0b938081e033233bae5569ae3b9fe8cb/packages/core/src/sdJwtVc/sdJwtVc.ts#L41

Reply:
Oh, That's good idea.